oat-sa/extension-tao-itemqti
# Fix/TR-1131/Update the size of the choice interaction once the theme has been applied to the item

Related to: https://oat-sa.atlassian.net/browse/TR-1131

Requires: 
 - [x] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/187
 - [x] https://github.com/oat-sa/tao-core/pull/2911
 - [x] once the companion PR has been merged, the package.json and package-lock.json files must updated with the proper version
 - [x] once the PR on tao-core has been merged, the dependency version must be updated in composer.json

Apply a resize of the choice interaction once the item them has been applied.

The issue reproduces only in Chrome as the load event is not emitted for the LINK elements, so that we need to rely on a different approach to adapt the size once the theme is applied.

How to test:
- make sure to have the branches applied both on tao-core and tao-itemqti
- have a test with horizontal choice and a custom theme (you may export the test linked in the ticket)
- navigate to the item and check the size of the choices, they should not be huge (check the ticket for more details)
